docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: '2.1'
+
+services:
+
+  base-etl:
+    build:
+      context: .
+    ports:
+      - 80:80
+    volumes:
+      - .:/code
+    environment:
+      SYNDICATION_API_KEY: ${SYNDICATION_API_KEY}
+      ETL_SCHEDULE: ${ETL_SCHEDULE}
+
+  full-etl:
+    extends:
+      service: base-etl
+    volumes:
+      - etl-node_modules:/code/node_modules
+
+  partial-etl:
+    extends:
+      service: base-etl
+    command: yarn run start-small
+    volumes:
+      - etl-node_modules:/code/node_modules
+
+  test-etl:
```

Comment:
The other apps have the test compose file separate from the running compose file. Is there a reason that is not the case?